demo/src/index.tsx

```diff
@@ -3,10 +3,14 @@ import { createRoot } from "react-dom/client";
 import App from "./App";
 import "@picocss/pico";
 import "./css/styles";
+("use client");
```

Comment:
I don't think this is necessary in this case? We aren't doing any server-side rendering